shared/router-v2/router.desktop.tsx

```diff
@@ -32,29 +32,6 @@ const routesMinusRoots = (tab: DesktopTabs) => {
   }, {})
 }
 
-// we must ensure we don't keep remaking these components
-const tabScreensCache = new Map<DesktopTabs, ReturnType<typeof makeNavScreens>>()
```

Comment:
move this into a memo so it gets clear on unmount, also we dont need this inner cache as the outer call it memod so it only runs once ever